packages/axelar-wasm-std/src/address.rs

```diff
@@ -43,6 +44,19 @@ pub fn validate_address(address: &str, format: &AddressFormat) -> Result<(), Err
             ScAddress::from_str(address)
                 .change_context(Error::InvalidAddress(address.to_string()))?;
         }
+        AddressFormat::Base58Solana => {
+            const SOLANA_PUBKEY_LEN: usize = 32;
+            const MAX_BASE58_LEN: usize = 44;
+            if address.len() > MAX_BASE58_LEN {
+                bail!(Error::InvalidAddress(address.to_string()));
+            }
+            let pubkey_vec = bs58::decode(address)
+                .into_vec()
+                .change_context(Error::InvalidAddress(address.to_string()))?;
+            if pubkey_vec.len() != SOLANA_PUBKEY_LEN {
+                bail!(Error::InvalidAddress(address.to_string()))
+            }
```

Comment:
```suggestion
            const SOLANA_PUBKEY_LEN: usize = 32;
            const MAX_ADDRESS_LEN: usize = 44;

            ensure!(address.len() <= MAX_ADDRESS_LEN, Error::InvalidAddress(address.to_string()));

            let pubkey = bs58::decode(address)
                .into_vec()
                .change_context(Error::InvalidAddress(address.to_string()))?;

            ensure!(pubkey_vec.len() == SOLANA_PUBKEY_LEN, Error::InvalidAddress(address.to_string()));
```
add newlines and use ensure for brevity

Reply:
Updated in https://github.com/axelarnetwork/axelar-amplifier/pull/1061/commits/e860f425ad3c63895cc33b9252ad44fd9aa7f80b